### PR TITLE
Add CPANSA-GitLab-API-v4-2023-01 for GitLab::API::v4

### PR DESCRIPTION
**Pull request by stigtsp**
CVE requested from MITRE

### CODE REVIEW

**Review by briandfoy — APPROVED** [review]

File: cpansa/CPANSA-GitLab-API-v4.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+- id: CPANSA-GitLab-API-v4-2023-01
```
Comment (briandfoy) [inline cpansa/CPANSA-GitLab-API-v4.yml:2]: This needs the affected_versions key.

**Review by briandfoy — CHANGES_REQUESTED** [review]

File: cpansa/CPANSA-GitLab-API-v4.yml (hunk 1 above)
Comment (stigtsp) [reply cpansa/CPANSA-GitLab-API-v4.yml:2]: added `affected_versions: ~`

File: cpansa/CPANSA-GitLab-API-v4.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  affected_versions: ~
+  fixed_versions: ~
+  description: >
+    GitLab::API:v4 is missing the verify_SSL=>1 flag in HTTP::Tiny, allowing a
```
Comment (garu) [inline cpansa/CPANSA-GitLab-API-v4.yml:6]: @stigtsp I think you missed a ':', should be '::v4', right?

File: cpansa/CPANSA-GitLab-API-v4.yml (hunk 1 above)
Comment (garu) [reply cpansa/CPANSA-GitLab-API-v4.yml:2]: @stigtsp I'm also not a fan of it being null. Even in the JSON Schema we created we stated that `affected_versions` must always be defined.

So, if it affects *only* the current version, I think it should say `"=0.26"` and we should have some sort of mechanism to monitor future releases to see if there was a fix; or, maybe simpler, just make a blank statement of `">= 0.26"` and let the author contact CPANSA about having fixed the code so we can update the entry to either a single release or a range.

But if it affects *all* versions, I think it should say `">0"`, maybe?

Comment (briandfoy) [reply cpansa/CPANSA-GitLab-API-v4.yml:2]: Yes, if it is still a problem, find the earliest version where it is a problem and use `>=` with no upper range.

**Review by briandfoy — APPROVED** [review]

**Pull request merged**